vyxal/elements.py

```diff
@@ -3361,6 +3361,17 @@ def newline_split(lhs, ctx):
     }.get(vy_type(lhs), lambda: vectorise(newline_split, lhs, ctx=ctx))()
 
 
+def next_multiple(lhs, rhs, ctx):
+    """Element ∆*
+    (num, num) -> get the next multiple of b that is greater than a
+    """
+
+    ts = vy_type(lhs, rhs)
+    return {
+        (NUMBER_TYPE, NUMBER_TYPE): lambda: lhs + rhs - lhs % rhs,
```

Comment:
Should this be nsimplified?

Reply:
I don't think it needs to be. The inputs given should already be nsimplified, no need to redo it